src/mac.c

```diff
@@ -185,3 +185,192 @@ static int handle_lbt_mode(struct nl802154_state *state,
 }
 COMMAND(set, lbt, "<1|0>",
 	NL802154_CMD_SET_LBT_MODE, 0, CIB_NETDEV, handle_lbt_mode, NULL);
+
+static int print_association_confirm_handler(struct nl_msg *msg, void *arg)
+{
+	int r;
+
+	uint16_t assoc_short_addr;
+	uint8_t status;
+
+	struct genlmsghdr *gnlh;
+	struct nlattr *tb[ NL802154_ATTR_MAX + 1 ];
+	int i;
+
+	printf( "in %s\n", __FUNCTION__ );
```

Comment:
Remove unnecessary comments like /* PAGE */. The code is / should be self-explanitory.